up.go

```diff
@@ -61,9 +62,28 @@ import (
 // TODO: [LATER] make it more friendly to infrequent Linux users by providing "descriptive" commands like "search" etc.
 // TODO: [LATER] advertise on: HN, r/programming, r/golang, r/commandline, r/linux, up-for-grabs.net; data exploration? data science?
 
+// Flag Settings
+var (
+	debugMode bool
+)
+
+func init() {
+	flag.BoolVar(&debugMode, "debug", false, "debug mode")
+	flag.BoolVar(&debugMode, "d", false, "debug mode (shorthand)")
+}
```

Comment:
Please make it:

```
var (
    debugMode = pflag.Bool("debug", false, "...")
)
```

and adjust the code to use `*debugMode` appropriately.

I'm not sure yet if debug mode should be needed by people so often to already validate steal one of the potentially valuable one-letter shortcuts...

Thanks!